guides/api_overview.md

```diff
@@ -311,3 +311,7 @@ You can implement the following callbacks:
 You will receive content share metrics if you registered a metric observer by [9.Receiving metrics (optional)](https://github.com/aws/amazon-chime-sdk-ios/blob/master/guides/api_overview.md#9-receiving-metrics-optional).
 
 Content share metrics will be prefixed by `contentShare`.
+
+## 14. Configuring Remote Video Subscriptions
+
+Amazon Chime SDK allows builders to have complete control over the remote videos received by each of their application’s end-users. This can be accomplished using the API [AudioVideoFacade.updateVideoSourceSubscriptions](https://aws.github.io/amazon-chime-sdk-android/amazon-chime-sdk/com.amazonaws.services.chime.sdk.meetings.audiovideo/-default-audio-video-facade/update-video-source-subscriptions.html). See [Configuring Remote Video Subscriptions](/guides/configuring_remote_video_subscriptions.md) for more information.
```

Comment:
need to update links. maybe grep for `android`

Reply:
Ah. Yeah, forgot to change the links when grep from android. Fixed.